sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -663,6 +665,75 @@ def process(self, context):
     self.check_label(beam.ParDo(MyDoFn()), r'ParDo(MyDoFn)')
 
 
+class PTransformTestDisplayData(unittest.TestCase):
```

Comment:
In Java, we also wrote some "integration" tests to ensure that display data gets properly attached to a pipeline job. There are many ways this can fail that aren't captured in unit tests: 

* Transforms will often wrap or adapt their components, and must ensure that the display data is also properly wrapped.
* Runners can replace transforms in a pipeline graph and must provide equivalent display data in the replacement.
* Composite transforms are lots during pipeline translation and thus composite display data must be passed down into leaf transforms.

In Java SDK, we selected a small subset of "important" transforms to write display data integration tests for-- mostly IO connectors. See [`ReadTest.testBoundedPrimitiveDisplayData()`](https://github.com/apache/incubator-beam/blob/master/sdks/java/core/src/test/java/org/apache/beam/sdk/io/ReadTest.java#L110)